src/redux/boards/actions/update.ts

```diff
@@ -0,0 +1,59 @@
+import { Dispatch } from '../../RootReducer'
+import { API } from '../../../services/http'
+
+import { IBoard } from '../types'
+
+export const UPDATE_BOARD = 'UPDATE_BOARD'
+export const UPDATE_BOARD_TITLE = 'UPDATE_BOARD_TITLE'
+export const UPDATE_BOARD_SUCCESS = 'UPDATE_BOARD_SUCCESS'
+export const UPDATE_BOARD_ERROR = 'UPDATE_BOARD_ERROR'
+
+export type Actions = {
+    UPDATE_BOARD: {
+        type: typeof UPDATE_BOARD
+    },
+    UPDATE_BOARD_ERROR: {
+        type: typeof UPDATE_BOARD_ERROR,
+        error: string
+    },
+    UPDATE_BOARD_SUCCESS: {
+        type: typeof UPDATE_BOARD_SUCCESS,
+        board: IBoard
+    }
+    UPDATE_BOARD_TITLE: {
```

Comment:
Why not use UPDATE_BOARD ?

Reply:
As discussed, it works like that, so I'm ok with it

Reply:
Ok :)